Source/WebCore/platform/audio/gstreamer/AudioEncoderGStreamer.cpp

```diff
@@ -56,7 +56,7 @@ class GStreamerInternalAudioEncoder : public ThreadSafeRefCountedAndCanMakeThrea
     String initialize(const String& codecName, const AudioEncoder::Config&);
     void postTask(Function<void()>&& task) { m_postTaskCallback(WTFMove(task)); }
     bool encode(AudioEncoder::RawFrame&&);
-    void flush(Function<void()>&&);
+    void flush();
```

Comment:
we now have two flush() methods, one returning a Ref<GenericPromise> and the other returning void.

Reply:
Yeah will fix this.